src/providers/storage/azureBlobStorage.ts

```diff
@@ -138,13 +141,41 @@ export class AzureCloudStorageService implements IStorageProvider {
             path = path ? [this.options.containerName, path].join("/") : this.options.containerName;
         }
         const files = await this.listFiles(path);
-        return files.map((blobPath) => AssetService.createAssetFromFilePath(blobPath));
+        const result: IAsset[] = [];
```

Comment:
Any tests needed to get created or updated for these changes?

Reply:
I've been playing around with this, I think I'll add them once the new stuff is in. That way it'll more accurately reflect the new data structure

Reply:
nvm, almost done adding tests now